syntax = "proto3";


message StatusFrame {
    bool kicked_off = 1;
    bytes extra = 2;
}